lms/djangoapps/instructor_task/api.py

```diff
@@ -437,14 +437,18 @@ def submit_export_ora2_data(request, course_key):
     return submit_task(request, task_type, task_class, course_key, task_input, task_key)
 
 
-def generate_certificates_for_students(request, course_key, students=None):  # pylint: disable=invalid-name
+def generate_certificates_for_students(request, course_key, student_set, specific_student):  # pylint: disable=invalid-name
     """
     Submits a task to generate certificates for given students enrolled in the course or
     all students if argument 'students' is None
 
     Raises AlreadyRunningError if certificates are currently being generated.
     """
-    if students:
```

Comment:
nit: I think we should throw an error if both `student_set` and `specific_student` are defined.